.circleci/config.yml

```diff
@@ -155,6 +156,30 @@ jobs:
           fail-on-issues: << parameters.snyk-fail-build >>
           additional-arguments: << parameters.snyk-args >>
 
+  assemble:
+    executor: hmpps/java
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gradle-{{ checksum "build.gradle.kts" }}
+            - gradle-
+      - run: ./gradlew assemble --console=plain
```

Comment:
what does --console=plain do?

Reply:
Ah, it's a leftover (I got into the habit of using it instead of the fancy outputs)

It "disables all color and other rich output in the console output. This is the default when Gradle is not attached to a terminal."

As it's the default on CircleCI, I'll remove it

Reply:
Removed in 9c97a137bae96a32e9f71e51d86810087fcb877b